nestkernel/conn_builder.cpp

```diff
@@ -386,7 +386,8 @@ nest::ConnBuilder::connect()
     && not symmetric_ && not is_symmetric() )
   {
     throw BadProperty(
-      "This synapse model requires symmetric (or all-to-all) connections" );
+      "This synapse model requires symmetric (or suitable uniform "
+      "all-to-all) connections" );
```

Comment:
"suitable" is unclear. How about "Connections with synapse model {0} can only be created as one-to-one connections with "make_symmetric" set to true or as all-to-all connections with equal source and target populations and default or scalar parameters."?